package.json

```diff
@@ -44,6 +44,7 @@
     "workspaceContains:**/*.hcl"
   ],
   "main": "./out/extension",
+  "browser": "./out/web/extension",
```

Comment:
Nitpick: Have you considered making it either

```json
  "main": "./out/desktop/extension",
  "browser": "./out/web/extension",
```
or
```json
  "main": "./out/desktop",
  "browser": "./out/web",
```
just for readability/clarity to highlight the fact the logic in the other file is for desktop?

Reply:
I'd be OK with that except that the "desktop" also is used with Codespaces and the Remote extensions. Technically "node" is more accurate, but I think that sounds more confusing.

Reply:
Hmmm, I see - that's a good point, and I agree `node` doesn't sound much clearer. Feel free to leave it as is! 😅